### PR TITLE
Add -a, --all flag to garbage command

### PR DESCRIPTION
**Pull request by userzimmermann**
And here it is: The first `--all` flag according to https://github.com/rustwasm/twiggy/issues/109 :)

I also added some new `twiggy garbage` tests for the new flag, based on `fixtures/wee_alloc.wasm`, since `fixtures/garbage.wasm` unfortunately only produces 9 items, but `-n <max_items>` defaults to `10` :D

### CODE REVIEW

File: opt/definitions.rs [hunk 1]
```diff
@@ -540,11 +545,12 @@ impl Garbage {
...
     /// The maximum number of items to display.
     pub fn max_items(&self) -> u32 {
-        self.max_items
+        if self.all_items { u32::MAX } else { self.max_items }
     }
 
     /// Set the maximum number of items to display.
     pub fn set_max_items(&mut self, max: u32) {
         self.max_items = max;
+        self.all_items = false;
```
Comment (data-pup) [inline opt/definitions.rs]: Good catch here :smile_cat:

File: twiggy/tests/tests.rs [hunk 2]
```diff
@@ -429,6 +429,32 @@ test!(
...
+
+test!(
+    garbage_wee_alloc_all_json,
+    "garbage",
+    "./fixtures/wee_alloc.wasm",
+    "-f",
+    "json",
+    "-a"
+);
+
```
Comment (data-pup) [inline twiggy/tests/tests.rs]: Nice job adding these tests as well!

**Comment by data-pup** [comment]
LGTM! Nice work @userzimmermann! :smile_cat: I see you've also started out on #120, that's awesome! Unless @fitzgen has any other input, I think this can be merged.

**Comment by fitzgen** [comment]
> Unless @fitzgen has any other input, I think this can be merged.

If it gets your approval stamp, that is all I need to know :)

File: twiggy/tests/tests.rs (hunk 2 above)
Comment (userzimmermann) [reply twiggy/tests/tests.rs]: Testing is the most important ;)

**Comment by userzimmermann** [comment]
Thx for nice feedback @data-pup @fitzgen! And approving, merging, etc :) Just right after you merged, it came to my mind that the `...top_10...` expectations should be better named like `...default_10...` or so :D to better understand what they are actually about... What do you think? Worth some extra PR? ;)

**Pull request merged**